### PR TITLE
Add Amazon Bedrock support

### LINKED ISSUE #459: Add Amazon Bedrock support
Hi, thank you for developing this great tool!

I'm asking if it is possible to add Amazon Bedrock support to `pr-agent` (just out of personal interest). Amazon Bedrock is a fully managed service that offers a choice of high-performing foundation models such as Claude, Cohere, etc, that runs on AWS. Since `litellm` already [supports Bedrock](https://docs.litellm.ai/docs/providers/bedrock), I believe we can add the support with a little change.

If it's acceptable, I'm willing to submit a PR. Thanks!

### PR DESCRIPTION
**Pull request by tmokmss**
closes #459 

Added support for Anthropic Claude on Amazon Bedrock. I'll add some explanation in the review comments.

### CODE REVIEW

File: pr_agent/algo/utils.py [hunk 1]
```diff
@@ -290,6 +290,7 @@ def _fix_key_value(key: str, value: str):
 
 def load_yaml(response_text: str) -> dict:
     response_text = response_text.removeprefix('```yaml').rstrip('`')
+    response_text = response_text.strip().rstrip().removeprefix('{').removesuffix('}')
```
[the remark this answers is not in the document]
Comment (mrT23) [reply pr_agent/algo/utils.py:293]: this is problematic.

what if the message actually starts with '}' ?
Its not something that should come in any way with a yaml, so removing it is risky

[the remark this answers is not in the document]
Comment (mrT23) [reply pr_agent/algo/utils.py:293]: Move this line, as a seperate fallback, to 'try_fix_yaml' section.

but this is not good for claude in terms of claude-vs-openai comparison :-)

[the remark this answers is not in the document]
Comment (tmokmss) [reply pr_agent/algo/utils.py]: Hi thanks. I moved them to `try_fix_yaml`. Yea I think I'll have to optimize prompts for Claude :)

**Review by mrT23 — APPROVED** [review]

**Comment by mrT23** [comment]
thanks @tmokmss

**Pull request merged**